UnitTests/CommonTestUtils/MockExecutable.cs

```diff
@@ -98,10 +98,10 @@ public IProcess Start(ArgumentString arguments, bool createWindow, bool redirect
 
         private sealed class MockProcess : IProcess
         {
-            public MockProcess([CanBeNull] string output, int? exitCode = 0)
+            public MockProcess([CanBeNull] string output, int? exitCode = 0, [CanBeNull] string error = null)
```

Comment:
❔
```suggestion
            public MockProcess(string? output, int? exitCode = 0, string? error = null)
```

Reply:
Removed the JetBrains annotation. There are a few other uses remaining, mostly in test files